intro_to_r/intro_to_r.Rmd

```diff
@@ -66,7 +82,7 @@ boys to newborn girls, so he gathered the baptism records for children born in
 London for every year from 1629 to 1710.  We can take a look at the data by 
 typing its name into the console.
 
-```{r view-data, eval=FALSE}
+```{r view-data}
 arbuthnot
```

Comment:
One caution:  If you adopt a model where you clobber your original data when you use `mutate()` and its kin, then you can have students mess up the original data.  So it might not be a bad idea to use `data( )` to make sure you start with what you planned on starting with.